config/secrets.yml

```diff
@@ -19,6 +19,14 @@
 
 development:
   secret_key_base: aca23d1befc693835c554a57e4b980ca3cd1d74ea145e07fea6357e1038c672e748e4201bf427ef889053eb56436a76e145dc0167dd17ca55c6a841097d9cb9a
+  mailer_domain: <%= ENV["MAILER_DOMAIN"] %>
```

Comment:
Here, it is not important but do not forgot to add this at least at you production environment. You can use deftault env or import ones from dev in prod